GithubBrowserSample/app/src/androidTest/java/com/android/example/github/ui/repo/RepoFragmentTest.java

```diff
@@ -67,18 +69,19 @@
     private MutableLiveData<Resource<Repo>> repo = new MutableLiveData<>();
     private MutableLiveData<Resource<List<Contributor>>> contributors = new MutableLiveData<>();
     private RepoFragment repoFragment;
+    @Mock
     private RepoViewModel viewModel;
 
+    @Mock
     private FragmentBindingAdapters fragmentBindingAdapters;
+    @Mock
     private NavigationController navigationController;
 
 
     @Before
     public void init() {
+        MockitoAnnotations.initMocks(this);
```

Comment:
We can simply annotate test class with `@RunWith(MockitoJUnitRunner.class)` instead of using `MockitoAnnotations.initMocks(this)`

Reply:
good call out - I think this @Mock annotation with initMocks is a good improvement too